client_plugin/vmdk_plugin/sanity_test.go

```diff
@@ -16,6 +16,8 @@
 // VMDK Docker driver sanity tests.
```

Comment:
this test was left out with the tags .. so adding a tag for the test to control during the build time.